jekyll/_cci2/browser-testing.md

```diff
@@ -158,7 +162,39 @@ workflows:
 ```
 {% endraw %}
 
-For more detailed information about the Sauce Labs orb and how you can use the orb in your workflows, refer to the [Sauce Labs Orb](https://circleci.com/developer/orbs/orb/saucelabs/sauce-connect) page in the [CircleCI Orbs Registry](https://circleci.com/developer/orbs).
+For more detailed information about using the Sauce Connect orb in your workflows, refer to the [Sauce Connect Orb](https://circleci.com/developer/orbs/orb/saucelabs/sauce-connect) page in the [CircleCI Orbs Registry](https://circleci.com/developer/orbs).
```

Comment:
@KunalJain It looks like we still need to have this orb updated if there is anyone you can put us in contact with, I know we started this once but maybe ran into issues.
https://circleci.com/developer/orbs/orb/saucelabs/sauce-connect

Reply:
I am not sure which team owns the sauce-connect orb, but I will see if I can ferret it out. Thanks!

Reply:
Hi @KyleTryon -- sorry for the delay on this. I have spoken with a number of folks at Sauce Labs, and it seems the Sauce Connect Orb development team has disbanded and there are no plans at this time to devote resources to it. I would like to propose an EOL to the Sauce Connect orb in its current state and amend this doc update to remove the Sauce Connect section so we might move forward with the update related to saucectl. Does that sound reasonable?
cc: @KunalJain